tests/tests/standalone-email-addresses/standalone-email-addresses.html

```diff
@@ -1,3 +1,4 @@
+SKIP
```

Comment:
This one completely messes up rSt syntax by using a custom plain-text format as if it was rSt.

Also, it might be taken the "automatic email" detection a bit too far by requiring all RFC 5322 addresses to be correctly detected?

Reply:
Do you think we should remove this from the span parser? Or is it just the regex that's a bit too greedy? 

Please note that the span handling is on my list to refactor because it's still messy. Spans should be a list of inline nodes. This will allow us to do replacements with the new compiler, making the whole span thing a bit less magic. :-)

Reply:
Well, I'm not sure what the official spec says about which e-mailaddresses or websites are automatically detected, but I would suggest to only have a very simple regex for the 80% use-case.

In the very complex 20% cases, people can either wrap it in `<...>` to be detected as hyperlink or use `` `some@email.com <mailto:...>`_ `` for e-mails.

Reply:
For your information, my todo list for this repo at the moment looks like:

1. Finish #139 
2. Finish #16 
3. Fix definition list and `code-block-diff` functional tests (as I've contributed both features to the rst-parser, it's probably the most efficient if I would port those changes over to this repo)